x-pack/plugins/beats_management/public/pages/overview/index.tsx

```diff
@@ -45,7 +49,19 @@ class MainPageComponent extends React.PureComponent<AppPageProps, MainPagesState
   public render() {
     return (
       <PrimaryLayout
-        title="Beats"
+        title={
+          <EuiFlexGroup alignItems="center" gutterSize="m">
+            <EuiFlexItem grow={false}>{'Beats'}</EuiFlexItem>
```

Comment:
Don't think you _need_ the `{''}` here, `<EuiFlexItem grow={false}>Beats</EuiFlexItem>` should work